src/components/forums/MyComments.js

```diff
@@ -0,0 +1,126 @@
+import React, { useRef, useState, useEffect } from "react"
+import { useAuth } from "../../contexts/AuthContext"
+import { Link, useHistory } from "react-router-dom"
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import NavigationBar from '../common/NavigationBar';
+import Box from "@material-ui/core/Box";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const MyComments = () => {
+
+    const { currentUser } = useAuth();
+    const history = useHistory();
+    const [email, setEmail] = React.useState('');
+    const [userID, setUserID] = React.useState('');
+
+    const [myComments, setMyComments] = React.useState([]);
+
+    React.useEffect(() => {
+        setEmail(currentUser.email);
+        loaduserSearchByEmail(currentUser.email);
+        //loadGetForums(); 
+    }, []);
+
+    const loaduserSearchByEmail = (email) => {
+        callApiGetuserSearchByEmail(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetuserSearchByEmail = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        //console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
```

Comment:
may need to change your api methods to "GET" if they are just pulling data from the db. Not sure if its easier for you to do now, or to fix later. I have the formal API formality that Saf wants in the Network and NetworkProfile files, and their respective apis